docs/artefacts/onramps.md

```diff
@@ -318,6 +318,8 @@ onramp:
 
 ### rest
 
+**This source can be linked**
```

Comment:
can we stick with one naming here that is `onramp`? Just for now. IIRC we are going to do the great renaming once we have connectors.

Reply:
good point all updated!